### PR TITLE
Add event on container death

### PR DESCRIPTION
**Pull request by mheon**
First step of converting Wait over to use events

### CODE REVIEW

**Comment by mheon** [comment]
Gets us an event formatted like this:

```
2019-03-12 16:25:27.088574667 -0400 EDT container died 70f01ba82a02575d3efc3579ee5951f1b3b3383243b204a54c21989e4b2d41fa (image=docker.io/library/fedora:latest, name=pensive_payne)
```

**Comment by baude** [comment]
LGTM

File: libpod/events.go [hunk 1]
```diff
@@ -19,6 +19,19 @@ func (c *Container) newContainerEvent(status events.Status) {
 	}
 }
 
+// newContainerDiedEvent creates a new event for a container's death
+func (c *Container) newContainerDiedEvent(exitCode int32) {
```
Comment (rhatdan) [inline libpod/events.go]: Should we call this Exited rather then Died?

File: libpod/events/events.go [hunk 2]
```diff
@@ -58,6 +58,8 @@ const (
 	Commit Status = "commit"
 	// Create ...
 	Create Status = "create"
+	// Died indicates that a container's process died
+	Died Status = "died"
```
Comment (rhatdan) [inline libpod/events/events.go]: Exited.

**Comment by mheon** [comment]
I matched Docker for the name but wouldn't nine changing to exited

**Comment by TomSweeneyRedHat** [comment]
I prefer "exited" too, but am just concerned about someone expecting 'died" due to a script that they run.  If that's a valid concern, I'd leave it as died.
Otherwise LGTM

**Comment by rhatdan** [comment]
Lets go with Exited since this is what happened.  Exited whether through successful running or failure.

**Comment by mheon** [comment]
Updated

**Pull request merged**